packages/gasket-plugin-intl/README.md

```diff
@@ -35,9 +35,9 @@ module.exports = {
   files from (default: `locales`)
 - `outputDir` - (string) path of directory to build locale and manifest files to
   (default: `./build/locales`)
-- `assetPrefix` - (string) change the default path to `/_locale` endpoint by
+- `zone` - (string) change the default path to `/_locale` endpoint by
   adding a path prefix here. (default: ''). Used for setting up CDN support for
-  locale files. `next.assetPrefix` will be used be unless specified here.
+  locale files.
```

Comment:
But mention `zone` is the default pathing unless otherwise specified
```suggestion
  locale files. The `zone` confiig will be used be unless `intl.assetPrefix` is set.
```